workers/unity/Packages/com.improbable.gdk.tools/DownloadCoreSdk.cs

```diff
@@ -13,6 +13,26 @@ internal enum DownloadResult
         Error
     }
 
+    internal class PluginPostprocessor : AssetPostprocessor
+    {
+        static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAssets, string[] movedAssets, string[] movedFromAssetPaths)
```

Comment:
Initial approach was to trigger Compatibility update right after plugins are downloaded. Unfortunately, this didn't work well with initial launch.
Even better solution would've been an explicit import of plugins after download and setting compatibility after that. Unfortunately, that will require a full fledged solution that can communicate with download logic to see what plugins were actually downloaded.